app/views/alchemy/admin/pages/edit.html.erb

```diff
@@ -14,14 +14,14 @@
   </div>
   <div class="toolbar_spacer"></div>
   <% unless @page.layoutpage? %>
-  <div class="button_with_label">
-    <%= form_tag alchemy.visit_admin_page_path(@page), id: 'visit_page_form' do %>
-      <button class="icon_button" title="<%= Alchemy.t('Visit page') %>">
-        <%= render_icon('external-link-alt') %>
-      </button>
-      <label><%= Alchemy.t("Visit page") %></label>
-    <% end %>
-  </div>
+    <div class="button_with_label">
+      <%= form_tag alchemy.visit_admin_page_path(@page), id: 'visit_page_form' do %>
+        <%= button_tag class: 'icon_button', title: (Alchemy.t('Visit page') if @page.public?), disabled: !@page.public? do %>
```

Comment:
The title is not used as tooltip. It should be removed actually as it is redundant. The `label` below is used to build the tooltip.